Makefile

```diff
@@ -49,6 +49,10 @@ test:
 test-clean:
 	$(shell sudo rm -rf /etc/containers/certs.d/127.0.0.1:8089/)
 
+.PHONY: install-notation
+install-notation:
+	notation -v || (curl -Lo notation.tar.gz https://github.com/notaryproject/notation/releases/download/v0.7.1-alpha.1/notation_0.7.1-alpha.1_linux_amd64.tar.gz && sudo tar xvzf notation.tar.gz -C /usr/bin notation; rm notation.tar.gz)
```

Comment:
https://github.com/project-zot/zot/pull/269/files#diff-76ed074a9305c04054cdebb9e9aad2d818052b07091de1f20cad0bbac34ffb52R52

^ try following this model, wherein I create a hack/tools/bin dir and download there, so becomes independent on that binaries are on the host system.